extension/textext/asktext.py

```diff
@@ -158,9 +158,9 @@ def callback(event):
 
     if isinstance(error, (TexTextConversionError, TexTextCommandFailed)):
         if error.stdout:
-            add_section("Stdout: <small><i>(click to expand)</i></small>", error.stdout)
+            add_section("Stdout: <small><i>(click to expand)</i></small>", error.stdout.decode('ascii'))
```

Comment:
I would change encoding to `utf-8`. It's default on linux/macos, and it's superset of plain ascii

Reply:
Done!